StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Elements/InlineSignup/LinkInlineSignupView.swift

```diff
@@ -131,16 +156,17 @@ final class LinkInlineSignupView: UIView {
             elements.append(contentsOf: [checkboxElement, combinedEmailNameSection])
         case .checkboxWithDefaultOptIn:
             elements.append(contentsOf: [checkboxElement, defaultOptInElement, combinedEmailNameSection].compactMap { $0 })
-        case .textFieldsOnlyEmailFirst:
-            elements.append(contentsOf: [emailSection, phoneNumberSection, nameSection])
-        case .textFieldsOnlyPhoneFirst:
-            elements.append(contentsOf: [phoneNumberSection, emailSection, nameSection])
+        case .textFieldsOnlyEmailFirst, .textFieldsOnlyPhoneFirst:
+            elements.append(combinedEmailNameSection)
```

Comment:
Updated the text-field-only variants to also use the `combinedEmailNameSection`, albeit with different ordering depending on the exact mode. Making this change so that the `SectionElement` automatically applies the capsule shape.